workers/ohsome_quality_analyst/base/report.py

```diff
@@ -64,10 +65,18 @@ def as_feature(self) -> Feature:
         The properties of the Feature contains the attributes of all indicators.
         The geometry (and properties) of the input GeoJSON object is preserved.
         """
-        properties = {}
+        report_properties = {
+            "metadata": vars(self.metadata).copy(),
+            "result": vars(self.result).copy(),
+        }
+        report_properties["metadata"].pop("label_description", None)
+        # TODO: do we need a request url?
```

Comment:
If so, I would do this not in this function but rather in the api.py module itself (E.g. `create_empty_resonse()`)